tests/golden/chatgpt-basic.md

```
@@ -0,0 +1,51 @@
+
+Here is code:
+
+```python
+print('hi')
+```
+
+And a link: https://example.com
+
+## Model
+
+Reply with an inline footnote: [7] works.
```

markdownlint-cli2 (0.22.0)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)